common/scripts/babel-conditional-preprocess.js

```diff
@@ -62,6 +62,11 @@ exports.default = babelHelper.declare((_api, opts) => {
       //
       // As this only applies to TypeScript types, it is safe from a code-flow perspective: This does not enable any new
       // conditional business logic flows.
+
```

Comment:
Please make sure that the comments from line 53 to 64 stay with `TSType`. That's what they apply to. Feel free to move it inside the {} so that they don't get lost in the future.

Reply:
Thanks for bringing this up. I have moved around my logic below the TSType logic